Tests/Integration/IntegrationTestData.swift

```diff
@@ -46,64 +46,65 @@ let currentUser = PassageUserDetails(
 
 
 let appInfoValid = AppInfo(
-    id: "czLTOVFIytGqrhRVoHV9o8Wo",
+    allowedIdentifier: "both",
+    authFallbackMethodString: "magic_link",
+    authOrigin: "http://localhost:4173",
     ephemeral: false,
+    id: "czLTOVFIytGqrhRVoHV9o8Wo",
+    loginURL: "/",
     name: "passage-ios uat",
-    redirect_url: "/dashboard",
-    login_url: "/",
-    allowed_identifier: "both",
-    required_identifier: "both",
-    auth_origin: "http://localhost:4173",
-    require_email_verification: false,
-    require_identifier_verification: false,
-    session_timeout_length: 0,
-    public_signup: true
+    publicSignup: true,
+    redirectURL: "/dashboard",
+    requiredIdentifier: "both",
+    requireEmailVerification: false,
+    requireIdentifierVerification: false,
+    sessionTimeoutLength: 0
```

Comment:
I don't know if this is valid. I think you need to have a value greater than 60s?

Reply:
This PR only updated the property names in the `AppInfo` class, but I can update that specific test value if needed? I'll have to look into the purpose of this specific test.